ChangeLog.d/ecdsa-verify-fixes.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix ECDSA verification, where it was failing to validate the public key
+     for signatures where R == 1 and S == 1. This bug meant that it was
```

Comment:
Not just R=S=1, there are a few other corner cases. In the changelog we don't need to detail the corner cases.